### ISSUE TITLE
Should we stop autofilling description for IH

### ISSUE THREAD

**Issue by MortenHofft** [gbif/registry#450]
I understand that IH do not have descriptions.
But that our current practice is to concatenate various fields and put them in the description (not a required field). But only do that for the first import.

Perhaps we should rather not do that.
At best it is pointless repetition. At worst - and most likely - it goes stale.

**Comment by ManonGros** [comment]
I found that having information like "this herbarium was merged with this one" at the institution level was quite useful. And the description was as good place as any to show that.

But I am not against keeping the description empty if you think this is better.
@marcos-lg would it be possible to do that?

**Comment by MortenHofft** [comment]
I'm not sure what I think, partly because I'm not sure what the current rule is.

**Comment by marcos-lg** [comment]
We never fill the description in the IH sync. The only thing that we set only in the first import is the collection name because by default we set is as `Herbarium` but we allow editors to modify it afterwards. 

We can see what fields are synced in these endpoints:
https://api.gbif.org/v1/grscicoll/institution/sourceableFields
https://api.gbif.org/v1/grscicoll/collection/sourceableFields

And there is a field `overridable` that means that the field can be overridden by the editors. This is the case only for the collection name.

I remember that at some point we talked about concatenating some fields but I think we discarded that option and never did it.
